src/screens/ProfileScreen/index.js

```diff
@@ -113,10 +118,27 @@ const ProfileScreen = () => {
                 </div>
             </div>
             <div className="col-12 m-5 pt-3 ps-5 pe-5 nav-pills fs-4">
-                Navbar
+                <div className={"row gx-5"}>
+                    <Link to=""
+                          className={`col text-center nav-link ${location.pathname.match(/profile$/) ? "active" : ""}`}>
+                        {profileOwner.username}'s Profile</Link>
+                    <Link to="lists"
+                          className={`col text-center nav-link ${location.pathname.match(/profile\/lists/) ? "active" : ""}`}>
+                        {profileOwner.username}'s Lists</Link>
```

Comment:
No need to repeat the username in the nav tabs. Just Profile, Lists, Reviews, Likes would be enough.